package/element/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "atropos/element",
+  "private": true,
+  "type": "module",
+  "exports": {
+    "./element": "./element/atropos-element.esm.js",
```

Comment:
These file paths are wrong, no `./element` here as we are already in this folder